testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -1218,3 +1219,60 @@ def test_record_type_sel():
 
     assert len(u.select_atoms('name CA and not record_type HETATM')) == 30
     assert len(u.select_atoms('name CA and record_type HETATM')) == 2
+
+
+def test_element_sel():
+    # test auto-topattr addition of string
+    u = mda.Universe(PDB_elements)
+    assert len(u.select_atoms("element H")) == 8
+    assert len(u.select_atoms("same element as index 12")) == 8
+
+
+def test_chain_sel():
+    u = mda.Universe(PDB_elements)
+    assert len(u.select_atoms("chainID A")) == len(u.atoms)
+
+
+@pytest.mark.parametrize("selstr,n_atoms", [
+    ("mass 0.8 to 1.2", 23850),
+    ("mass 8e-1 to 1200e-3", 23850),
+    ("mass -5--3", 2),  # select -5 to -3
+    ("mass -3 : -5", 0),  # wrong way around
+])
+def test_mass_sel(selstr, n_atoms):
+    # test auto-topattr addition of float (FloatRangeSelection)
+    u = mda.Universe(TPR)
+    u.atoms[-10:].masses = - (np.arange(10) + 1.001)
+    assert len(u.select_atoms(selstr)) == n_atoms
+
+
+@pytest.mark.parametrize("selstr,n_res", [
+    ("resnum -10 to 3", 14),
+    ("resnum -5--3", 3),  # select -5 to -3
+    ("resnum -3 : -5", 0),  # wrong way around
+])
+def test_int_sel(selstr, n_res):
+    # test auto-topattr addition of int (IntRangeSelection)
+    u = mda.Universe(TPR)
+    u.residues[-10:].resnums = - (np.arange(10) + 1)
+    ag = u.select_atoms(selstr).residues
+    assert len(ag) == n_res
+
+
+def test_bool_sel():
+    pytest.importorskip("rdkit.Chem")
+    u = MDAnalysis.Universe.from_smiles("Nc1cc(C[C@H]([O-])C=O)c[nH]1")
+    assert len(u.select_atoms("aromaticity")) == 5
```

Comment:
I don't like this selection name, is this what it is already called? I think 'aromatic'/'not aromatic' is correct

Reply:
It's because "aromaticity" is the name of the topology attribute and it just gets automatically added. I think it's the only Boolean selection we have right now, which is why it's in the test. There is also a separately defined topology attribute with the "aromatic" token